open-source.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Minimum Open Source requirements for Services
+description: The minimum bar for Open Source when working on Services, that then allows further progression.
```

Comment:
```suggestion
description: The minimum requirements for Open Source when working on Services, that then allows further progression.
```
- `bar` is a more of an idiomatic phrase and less clear, more difficult to translate.
- I have come around to agreeing that although it looks awkwardly Germanic, we should be capitalizing Open Source as part of controlling the narrative around what Open Source means. I'm not sure though about `Services` being capitalized unless you also define what this special instance of the word services means.

Reply:
Indeed my German first language is creeping in here. I've removed capitalization from the word Service, except in headings where the heading is not a full sentence.